relayer/chains/wasm/provider.go

```diff
@@ -70,10 +70,10 @@ func (p *Provider) Init(ctx context.Context, homePath string, kms kms.KMS) error
 
 // Wallet returns the wallet of the provider
 func (p *Provider) Wallet() sdkTypes.AccAddress {
```

Comment:
`sdkTypes.AccAddress` does not have signer capability. Shouldn't a wallet have signing capability? I think the type for wallet should be `keyring.Keyring`. And this keyring should be injected while creating client context.

Reply:
We do not indent to use signer here.

Reply:
In that case, the function name `Wallet()` and the field name `wallet` in provider type is misleading.